### PR TITLE
Feature/Group/CLK32-906-Group-Detail

### PR DESCRIPTION
**Pull request by Minhdn1**
## Prerequisite
- [x] Self-review :monocle_face:
- [x] No Crashes ☠️
---

### Root cause
- UI Screen Group Detail

### Solution
- // Explain how you fix it

### Impact
- UI Screen Group Detail

### Screenshots (Optional)

- Light

### CODE REVIEW

**Comment by namnh92** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

- [ ] I understand that "General.Notification" is now existed in Localized service

- [ ] I understand that "Home.Profile" is now existed in Localized service

- [ ] I understand that "Home.Sever" is now existed in Localized service

- [ ] I understand that "General.Next" is now existed in Localized service

- [ ] I understand that "GroupDetail.AddMember" is now existed in Localized service

- [ ] I understand that "GroupDetail.Search" is now existed in Localized service

- [ ] I understand that "GroupDetail.Title.PlaceHoder" is now existed in Localized service

- [ ] I understand that "GroupDetail.TitleCheckbox" is now existed in Localized service

- [ ] I understand that "General.Audio" is now existed in Localized service

- [ ] I understand that "General.Video" is now existed in Localized service

- [ ] I understand that "GroupDetail.RemoveMember" is now existed in Localized service

- [ ] I understand that "GroupDetail.SeeMembers" is now existed in Localized service

- [ ] I understand that "Home.Signout" is now existed in Localized service

- [ ] I understand that "General.Search" is now existed in Localized service

- [ ] I understand that "GroupDetail.UserInitTitle" is now existed in Localized service

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: ClearKeep/App/Theme/AppImageSet.swift [hunk 1]
```diff
@@ -68,7 +68,7 @@ struct AppImageSet: IAppImageSet {
 	var googleIcon: Image { Image("ic_google") }
 	var officeIcon: Image { Image("ic_office") }
 	var facebookIcon: Image { Image("ic_faceBook") }
-	var checkMask: Image { Image(systemName: "checkmark.circle") }
+	var checkMask: Image { Image(systemName: "circle.fill") }
```
Comment (namnh92) [inline ClearKeep/App/Theme/AppImageSet.swift]: @Minhdn1 : Please get icon from figma

File: ClearKeep/App/Theme/AppImageSet.swift [hunk 2]
```diff
@@ -68,7 +68,7 @@ struct AppImageSet: IAppImageSet {
...
 	var checkMaskFill: Image { Image(systemName: "checkmark.circle.fill") }
```
Comment (namnh92) [inline ClearKeep/App/Theme/AppImageSet.swift]: @Minhdn1 : Please get icon from figma

**Review by namnh92 — CHANGES_REQUESTED** [review]

File: ClearKeep/App/Theme/AppImageSet.swift (hunk 1 above)
Comment (Minhdn1) [reply ClearKeep/App/Theme/AppImageSet.swift]: fix complete

File: ClearKeep/App/Theme/AppImageSet.swift (hunk 2 above)
Comment (Minhdn1) [reply ClearKeep/App/Theme/AppImageSet.swift]: fix complete

**Review by namnh92 — APPROVED** [review]

**Pull request merged**